.github/workflows/nightly-build.yml

```
@@ -66,6 +66,10 @@ env:
 
   NIGHTLY_RELEASE_PREFIX: nightly
 
+  # Use the different image name to avoid conflict with the release images.
+  # The DockerHub image will be greptime/greptimedb-nightly.
+  IMAGE_NAME: greptimedb-nightly
```

```suggestion
  IMAGE_NAME: greptimedb-weekly
```

Maybe we can distinguish two nightly image as "weekly" and "daily" according to their natural property?

---

Make sense. It should use `daily` in this PR. Let me change it.

---

@tisonkun I just thought about it more carefully. I think we can:

- `nightly-build.yaml` -> use `greptime/greptimedb-nightly:nightly-xxx` because it's nightly, actually;
- scheduled weekly release -> use `greptime/greptimedb:v0.2.0-weekly-20230313`;

It's more precise and easy to classify. Of cause, we also should add the documents to describe our CD strategy.